FirebaseMessaging/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# unreleased
```

Comment:
If it merges today or tomorrow, it will be 8.2.0

Reply:
Done.